fossunited/chapters/web_template/foss_club___criteria_for_lead/foss_club___criteria_for_lead.html

```diff
@@ -23,12 +23,12 @@
   }
 
   h2 {
-    margin: 0;
-    color: #000;
-    font-size: 1.5rem;
-    font-weight: 700;
-    line-height: 100%; /* 1.5rem */
-    letter-spacing: -0.06rem;
+  margin: 0;
+  color: #000;
+  font-size: 1.5rem; 
```

Comment:
```suggestion
  font-size: 1.5rem;
```

removed trailing whitespace